app/src/main/res/layout/activity_login.xml

```diff
@@ -124,116 +126,118 @@
     <Button
         android:id="@+id/btn_login"
         android:layout_width="match_parent"
-        android:layout_height="49dp"
-        app:layout_constraintTop_toBottomOf="@id/txt_forget"
+        android:layout_height="@dimen/main_activity_height_input_btn"
+        android:layout_marginTop="@dimen/main_activity_margin_top_btn"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks